spec.md

```diff
@@ -304,6 +317,26 @@ Location: <blob-location>
 Here, `<blob-location>` is a pullable blob URL.
 
 
+##### Mounting a blob from another repository
+If a necessary blob exists already in another repository, it can be mounted into a different repository via a `POST`
+request in the following format:
+
+`/v2/<name>/blobs/uploads/?mount=<digest>&from=<repository_name>`  <sup>[end-11](#endpoints)</sup>.
+
+In this case, `<name>` is the namespace to which the blob will be mounted. `<digest>` is the digest of the blob to mount,
+and `<repository_name>` is the namespace from which the blob should be mounted. This step is usually taken in place of the
+previously-described `POST` request to `/v2/<name>/blobs/uploads/` <sup>[end-4a](#endpoints)</sup> (which is used to initiate an
+upload session).
+
+The response to a successful mount MUST be `201 Created`, and MUST contain the following header:
+```
+Location: <blob-location>
+```
+
+
+Alternatively, if a registry does not support cross-repository mounting, it SHOULD return a `202`, indicating that the
```

Comment:
> if a registry does not support cross-repository mounting

It's possible that the registry supports cross-repository mounting but is unable to mount the requested blob for various reasons (malformed request or blob non-existence, random distributed systems failures, etc.). May say something like "or is unable to mount the requested blob".